### PR TITLE
Bem

### PR DESCRIPTION
**Pull request by shtbik1**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,23 +1,303 @@
...
+      <p class="title">
+        From beginning ideas to individual integrity, <br />
+        rich identity from the line
+        <span class="title__text_colorpurple">
+          on the paper to <br />
+          final projects
+        </span>
+      </p>
+      <div class="specification" id="specification">
+        <img src="/assets/image 11.png" alt="Стена" />
```
Comment (shtbik) [inline index.html]: Don't use names with spaces

File: index.html [hunk 2]
```diff
@@ -1,23 +1,303 @@
...
-        <div class="col">My site 4</div>
-        <div class="col">My site 5</div>
-        <h3 class="col">My site 6</h3>
-    </div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles/styles.css" />
+    <title>Document</title>
```
Comment (shtbik) [inline index.html]: Add real title and description

File: index.html [hunk 3]
```diff
@@ -1,23 +1,303 @@
...
+          <hr class="profile__about__underLogo" />
+          <p class="profile__about__text_smallText">
+            I am currently open for a part-time work
+          </p>
+        </div>
+        <div class="profile__profileimage">
+          <img
+            class="profile__profileimage__img"
+            src="/assets/facef.png"
+            alt="Фото 4х3"
```
Comment (shtbik) [inline index.html]: Use english

File: index.html [hunk 4]
```diff
@@ -1,23 +1,303 @@
...
+            <span
+              ><p>
+                Mobile application development is a highlight that <br />
+                businesses are interested in at the moment and in the <br />
+                future.
+              </p></span
+            >
+            <a
+              class="specification__skills__links"
+              href="https://github.com/SHTbIK37"
```
Comment (shtbik) [inline index.html:110]: Add `target="_blank"` attribute to open the link in a new tab. Better to do it for all external links

File: index.html [hunk 5]
```diff
@@ -1,23 +1,303 @@
...
+        other inquiries.
+      </span>
+      <div class="footer__contacts">
+        <a href="#"><img src="/assets/twitter.png" alt="twitter" /></a>
+        <a href="#"><img src="/assets/facebook.png" alt="facebook" /></a>
+        <a href="#"><img src="/assets/mail.png" alt="mail" /></a>
+        <a href="#"><img src="/assets/instagram.png" alt="instagram" /></a>
+      </div>
+    </footer>
+    <!-- <script src="./js/script.js"></script> -->
```
Comment (shtbik) [inline index.html]: Remove

File: styles/styles.css [hunk 6]
```diff
@@ -0,0 +1,768 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+body {
+  background-color: white;
+  margin: 0;
+  font-family: "Inter", sans-serif;
+  align-items: center;
```
Comment (shtbik) [inline styles/styles.css:7]: Is it necessary?

File: styles/styles.css [hunk 7]
```diff
@@ -0,0 +1,768 @@
...
+  font-size: 18px;
+}
+.navigation {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  margin-top: 118px;
+  margin-right: 128px;
+}
+.links {
```
Comment (shtbik) [inline styles/styles.css:20]: Cool 👍🏼

File: styles/styles.css [hunk 8]
```diff
@@ -0,0 +1,768 @@
...
+  margin-right: 83px;
+}
+.navigation__blog {
+  margin-right: 79px;
+}
+.navigation__hire {
+  margin-right: -15px;
+  position: relative;
+  color: #3730a3;
+  padding: 13px 32px 13px 32px;
```
Comment (shtbik) [inline styles/styles.css]: You can use short standard:
`padding: 13px 32px`

File: styles/styles.css [hunk 9]
```diff
@@ -0,0 +1,768 @@
...
+
+body {
+  background-color: white;
+  margin: 0;
+  font-family: "Inter", sans-serif;
+  align-items: center;
+}
+
+header {
+  font-size: 18px;
```
Comment (shtbik) [inline styles/styles.css:11]: Remind me to tell you about a different types of dimensions - `px/rem/em`

File: styles/styles.css [hunk 10]
```diff
@@ -0,0 +1,768 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 150px;
+  margin-bottom: 147px;
+  background-color: #fef2f2;
+  color: #9ca3af;
+  font-size: 20px;
+  height: 300px;
```
Comment (shtbik) [inline styles/styles.css]: Are you sure that you need it?

File: styles/styles.css [hunk 11]
```diff
@@ -0,0 +1,768 @@
...
+}
+.trustedblock__img {
+  width: 67%;
```
Comment (shtbik) [inline styles/styles.css]: Check this block on the mobile view. If you use `width` in percentage, it's worth adding `min-width`.

File: styles/styles.css [hunk 12]
```diff
@@ -0,0 +1,768 @@
...
+}
+
+.imgproject__photos_link {
+  display: flex;
+  position: relative;
+}
+.imgproject__photos_link__text {
+  display: flex;
+  height: 92px;
+  width: 495px;
```
Comment (shtbik) [inline styles/styles.css:247]: This looks not flexible. What will happen on the mobile?

File: styles/styles.css [hunk 13]
```diff
@@ -0,0 +1,768 @@
...
+  align-items: center;
+}
+.imgproject__photos_link__text__link a {
+  display: flex;
+  align-items: center;
+}
+
+.planning {
+  margin: 0 328px;
+  margin-top: 150px;
```
Comment (shtbik) [inline styles/styles.css]: Replace above two lines with:
`margin: 150px 328px 0;`

File: styles/styles.css [hunk 14]
```diff
@@ -0,0 +1,768 @@
...
+  display: flex;
+  flex-direction: row;
+  font-weight: 600;
+}
+.planning__block__text_bold {
+  font-size: 30px;
+}
+.planning__block__text {
+  display: block;
+  font-weight: 400;
```
Comment (shtbik) [inline styles/styles.css:297]: This should be by default. You also can use the alias "normal"

File: styles/styles.css [hunk 15]
```diff
@@ -0,0 +1,768 @@
...
+}
+.careers__logo {
+  font-weight: 600;
+  text-align: center;
+  font-size: 48px;
+}
+.careers div {
+  display: flex;
+  justify-content: space-between;
+
```
Comment (shtbik) [inline styles/styles.css:317]: Can remove this line

File: styles/styles.css [hunk 16]
```diff
@@ -0,0 +1,768 @@
...
+  margin-top: 58px;
+  margin-left: 152px;
+}
+.careers__company,
+.careers__position,
+.careers__year {
```
Comment (shtbik) [inline styles/styles.css:323]: 👍🏼

File: styles/styles.css [hunk 17]
```diff
@@ -0,0 +1,768 @@
...
+    max-width: 100%;
+  }
+  .imgproject__photos img {
+    max-width: 100%;
+  }
+  .imgproject__photos_link__text {
+    max-width: 85%;
+  }
+
+  #prlinkimg {
```
Comment (shtbik) [inline styles/styles.css]: I think it's something old

File: styles/styles.css [hunk 18]
```diff
@@ -0,0 +1,768 @@
...
+    font-size: 14px;
+    margin: 1%;
+    margin-bottom: 5%;
+  }
+  footer img {
+    margin-bottom: 10%;
+    width: 100%;
+  }
+}
+@media only screen and (min-width: 250px) and (max-width: 550px) {
```
Comment (shtbik) [inline styles/styles.css]: You can remove `(min-width: 250px)`

File: styles/styles.css [hunk 19]
```diff
@@ -0,0 +1,768 @@
...
+  height: 100px;
+  width: 60%;
+  justify-content: space-around;
+  margin-top: 53px;
+  margin-bottom: 45px;
+}
+.footer__contacts a {
+  height: 35px;
+}
+@media only screen and (min-width: 769px) and (max-width: 1280px) {
```
Comment (shtbik) [inline styles/styles.css:385]: The idea is to accumulate styles when you get down so you don't need to to write the same styles every breakpoint. To make it you don't need restrict it from both sides (max-width and min-width), use max-width only:
default styles -> (max-width: 1280px) (and less) -> (max-width: 768px) (includes styles from previous 1280px and less) -> (max-width: 550px) (and so on) -> and less.

Look at this to get an idea
https://habr.com/ru/post/269419/

Don't need to change it this time but good to know for the future.

File: index.html [hunk 20]
```diff
@@ -1,23 +1,303 @@
 <!DOCTYPE html>
 <html lang="en">
```
Comment (shtbik) [inline index.html:2]: Use https://validator.w3.org/#validate_by_upload to check correctness of your HTML

**Review by shtbik — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (shtbik1) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (shtbik1) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (shtbik1) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (shtbik1) [reply index.html:110]: done

File: index.html (hunk 5 above)
Comment (shtbik1) [reply index.html]: done

File: styles/styles.css (hunk 9 above)
Comment (shtbik1) [reply styles/styles.css:11]: done

File: styles/styles.css (hunk 10 above)
Comment (shtbik1) [reply styles/styles.css]: yes

File: styles/styles.css (hunk 11 above)
Comment (shtbik1) [reply styles/styles.css]: on mobile looks good

File: styles/styles.css (hunk 12 above)
Comment (shtbik1) [reply styles/styles.css:247]: for different resolution i use different width

File: styles/styles.css (hunk 13 above)
Comment (shtbik1) [reply styles/styles.css]: done

File: styles/styles.css (hunk 14 above)
Comment (shtbik1) [reply styles/styles.css:297]: ok

File: styles/styles.css (hunk 17 above)
Comment (shtbik1) [reply styles/styles.css]: done

File: index.html (hunk 20 above)
Comment (shtbik1) [reply index.html:2]: pass

File: styles/styles.css (hunk 6 above)
Comment (shtbik1) [reply styles/styles.css:7]: yes

File: styles/styles.css (hunk 8 above)
Comment (shtbik1) [reply styles/styles.css]: done

File: styles/styles.css (hunk 18 above)
Comment (shtbik1) [reply styles/styles.css]: done

**Comment by shtbik** [comment]
Review responsive design again and try to make it better:

Снимок экрана 2022-06-09 в 8 05 43 PM
Снимок экрана 2022-06-09 в 8 06 33 PM
Снимок экрана 2022-06-09 в 8 04 22 PM
Снимок экрана 2022-06-09 в 8 04 00 PM
Снимок экрана 2022-06-09 в 8 04 49 PM
Снимок экрана 2022-06-09 в 8 05 38 PM

**Review by shtbik — APPROVED** [review]
g2g

**Pull request merged**